radicle/src/storage/git.rs

```diff
@@ -400,14 +368,33 @@ impl Repository {
         Ok(remotes)
     }
 
-    pub fn sign_refs<G: Signer>(&self, signer: G) -> Result<SignedRefs<Verified>, Error> {
-        let remote = signer.public_key();
-        let refs = self.references(remote)?;
-        let signed = refs.signed(&signer)?;
+    /// Return all references that are public, ie. that are replicated, verified and signed.
```

Comment:
nit: The public doesn't fit right with me. It's more that you're returning a filtered set of `namespaced_references`, right? The canonical references are also public per se.

Reply:
5f4fc8236ee08ae279a1e09f033bfb5acddcf607